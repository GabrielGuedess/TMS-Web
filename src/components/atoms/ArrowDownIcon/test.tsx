import { render } from '@testing-library/react';

import { ArrowDownIcon } from '.';

describe('<ArrowDownIcon />', () => {
  it('should render the arrow down icon correctly', () => {
    const { container } = render(<ArrowDownIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
