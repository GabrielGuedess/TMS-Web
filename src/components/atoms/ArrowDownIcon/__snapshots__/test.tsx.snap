// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowDownIcon /> should render the arrow down icon correctly 1`] = `
<svg
  fill="none"
  height="10"
  viewBox="0 0 10 10"
  width="10"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M5.64284 7.69237L9.09102 4.33987L10 5.22362L5 10.0849L-8.98488e-07 5.22362L0.908973 4.33987L4.35716 7.69237L4.35716 0.0848701L5.64284 0.0848704L5.64284 7.69237Z"
    fill="currentColor"
  />
</svg>
`;
